code/modules/projectiles/projectile/special.dm

```diff
@@ -24,8 +24,9 @@
 /obj/item/projectile/bullet/rocket
 	name = "high explosive rocket"
 	icon_state = "rocket"
-	damage_types = list(BRUTE = 70)
+	damage_types = list(BRUTE = 50)
 	armor_penetration = 100
+	style_damage = 110 //single shot, incredibly powerful. If you get direct hit with this you deserve it, if you dodge the direct shot you're protected from the explosion.
```

Comment:
```suggestion
	style_damage = 100 //single shot, incredibly powerful. If you get direct hit with this you deserve it, if you dodge the direct shot you're protected from the explosion.
```
there's a difference between making it require you be at peak performance and not at all drained to dodge it, and ignoring the feature entirely.
any style_damage above 100 is the latter.

Reply:
Ignoring the feature entirely is intended. It is intended to ignore armor on a direct hit, and it will ignore style as well. You're still probably better off since style dodge will reduce explosion damage if it's an indirect hit. The same should frankly be applied to AMR IMO, but I'm not interested in touching AMR atm and it's less grievious to me to miss an AMR than an RPG shell.

Reply:
```suggestion
	style_damage = 101 //single shot, incredibly powerful. If you get direct hit with this you deserve it, if you dodge the direct shot you're protected from the explosion.
```
then maybe make it more obvious that it's unusual by making it 101?

Reply:
I mean I can set it to 101, I thought 110 would be more obvious than 101. 111 is probably the most obvious.